bin/ci

```diff
@@ -17,19 +17,46 @@ COMMAND:
 "
 }
 
+function usage_function() {
+    echo -e "usage: ${0} ${1} [--ignore-changelog | ignorechangelog] [-h | --help | help]
+
+${2}
+
+OPTIONS:
+  --ignore-changelog  does not include check-changelog job into workflows
+  --help              display this help message
+"
+}
+
 
 function check_changes() {
     to="${CIRCLE_SHA1:-HEAD}"
     git whatchanged --name-only --pretty="" origin/master.."${to}" | grep "${1}" &> /dev/null
 }
 
+# Utility function that updates circle configuration site-factory file according to
+# --ignore-changelog flag
+# Usage: update_factory [--ignore-changelog]
+function update_factory() {
+    disclaimer="# /!\ DO NOT EDIT: this file is autogenerated"
+    template=".circleci/src/workflows/site-factory.yml.tpl"
+    
+    # Add a workflow for the site from our template
+    printf "%s\n" "${disclaimer}" > ".circleci/src/workflows/site-factory.yml"
+
+    flags=""
+    if [ "$1" = true ] ; then
+        flags="--ignore-changelog"
+    fi
+
+    sed "s|\${CI-UPDATE-OPTIONS}|$flags|;" < $template >> ".circleci/src/workflows/site-factory.yml"
+}
 
 # Utility function that updates circle configuration source files with a
 # workflow for each site and running only the necessary jobs
 #
-# Usage: update_sources
+# Usage: update_sources [--ignore-changelog]
```

Comment:
```suggestion
# Usage: update_sources [-i | --ignore-changelog | ignore-changelog]
```

Reply:
Hmm in fact, `update_sources` just received a boolean value.

`# Usage: update_sources ignore-changelog<boolean>` ?

Reply:
true :thinking: ok!